frontend/src/components/FormRow.vue

```diff
@@ -66,7 +66,7 @@
                     <div class="block sm:inline sm:absolute sm:left-full sm:ml-4 mt-2 sm:mt-0"><slot name="append"></slot></div>
                 </template>
             </div>
-            <div v-if="error" class="float-right ml-4 text-red-400 inline text-xs">{{error}}</div>
```

Comment:
The intention for the float was so to stop the content below this row getting pushed down when the error is shown - not a fan of moving form elements.

Now you mention it, I do see the issue with the Project name row on the create page - with the recently added text about not being able to change name:

<img width="371" alt="Cursor_and_FlowForge" src="https://user-images.githubusercontent.com/51083/162235952-47265915-9e68-4163-9222-63316266e852.png">

In theory *think* we could apply `after:clear-both` to the top FormRow containing div - so it stops the float from overflowing the container... but I can't get that to work locally. Not going to sweat it - just wanted to highlight the reason for the float.

Reply:
There was another strange issue whereby the `float:right` was limiting the width of the `FormRow` directly after, if that `FormRow` was of `type="checkbox"`, was really strange as they had no reason to interact.